src/localisation/launch/ekf_node.launch

```diff
@@ -0,0 +1,36 @@
+<launch>
+    <node name="ekf_node" pkg="localisation" type="ekf_node" output="screen">
+	<!-- The error of the IMU angular velocity measurement -->
+        <rosparam param="sdev_gyro">0.017453292519943295</rosparam>
+
+	<!-- The error of the linear (forward) speed encoder measurement-->
+        <rosparam param="sdev_speed">0.15</rosparam>
+
+	<!-- The error of the gps x coordinate measurement -->
+        <rosparam param="sdev_gps_x_measurement">5.0</rosparam>
+
+	<!-- The error of the gps y coordinate measurement -->
+        <rosparam param="sdev_gps_y_measurement">5.0</rosparam>
+
+	<!-- The error of the IMU orientation measurement -->
+        <rosparam param="sdev_angle_measurement">0.017453292519943295</rosparam>
+
+	<!-- initialise covariance matrix -->
+        <rosparam param="p">[0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0]</rosparam>
+
+	<!-- matrix to account for unknown external influences -->
+        <rosparam param="q1">[0.0001, 0.0, 0.0, 0.0, 0.0001, 0.0, 0.0, 0.0, 0.000304617419786708]</rosparam>
+
+	<!-- observation matrix -->
+        <rosparam param="h">[1.0, 0.0, 0.0, 0.0, 1.0, 0.0, 0.0, 0.0, 1.0]</rosparam>
```

Comment:
For the covariance, unknown external influence, and observation matrix, could we add some basic instructions for how one might go about tuning them, and what each row/column in them is for? This is really about making it easier for people to use without having to go through and understand how kalman filters work.

Reply:
Yes please :+1:

Reply:
basic instructions have been added and variables have been added and deleted to make changing these matrices as easy as possible hopefully the descriptions are easy for user to understand